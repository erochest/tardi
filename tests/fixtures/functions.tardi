
// lambda
2 { 2 * } apply

// function
: triple 3 * ;

3 triple
dup triple

// TODO: add recursion once we have conditionals
