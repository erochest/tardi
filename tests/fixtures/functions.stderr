4
9
27
"is even"
"is not even"
"is not even"
"is even"
